src/cadenza.js

```diff
@@ -151,12 +151,15 @@ export class CadenzaClient {
    * @return {Promise<void>} A Promise for when the iframe is loaded
    * @throws For an invalid source
    */
-  show(source, { hideMainHeaderAndFooter, hideWorkbookToolBar, signal, accept } = {}) {
+  show(
+    source,
+    { hideMainHeaderAndFooter, hideWorkbookToolBar, signal, accept } = {},
```

Comment:
The parameter for download and fetch was renamed to mediaType, so that it's the same as the actual query parameter. (Don't know why it was different in the first place ...) Please align it here as well.

Reply:
I had overlooked the rename, I'll change it.